pkg/server/api/sharkey.go

```diff
@@ -86,6 +86,12 @@ func Run(conf *config.Config, logger *logrus.Logger) {
 		Handler:   loggingHandler,
 	}
 
+	if c.conf.Github.Enabled {
```

Comment:
I wonder if we should worry about a clean shutdown of the cron if the server is shutting down?

Reply:
Not sure if theres anything we can do here. Seems like the worst that could happen is that the db transaction is cancelled halfway through, but I think that should be rolled back by mysql/sqlite